cfn-resources/resource-policy/test/cfn-test-delete-inputs.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# cfn-test-delete-inputs.sh
+#
+# This tool deletes the mongodb resources used for `cfn test` as inputs.
+#
+
+set -o nounset
```

Comment:
is this necessary? could it be empty?

Reply:
Not necessary, forgot to delete the file. Usually this file is used to delete the created resources in Atlas, but in this case we are not creating any resource.